devicetypes/smartthings/wemo-switch.src/wemo-switch.groovy

```diff
@@ -16,7 +16,7 @@
  * Date: 2015-10-11
  */
  metadata {
- 	definition (name: "Wemo Switch", namespace: "smartthings", author: "SmartThings") {
+ 	definition (name: "Wemo Switch", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.smartplug") {
```

Comment:
Is this a smart plug or a switch? You used oic.d.smartplug here but the DTH says it's a switch.  Please verify.